### PR TITLE
City of Light General update

### PR DESCRIPTION
**Pull request by Kitsunemitsu**
## About The Pull Request
Added in the hana machine that prints money
Made minor mapping changes.
Butchers now have 60 in all stats to bring them just below Carnival.
Food removed from houses, buy junk food, cook your money, or buy from HHPP.
Injectors now give you 4/3/2/1 stats for 20/40/60/80+ to help players get to midgame better.

## Changelog
:cl:
balance: Hana can now print money.
balance: Injectors are now better below 80s.
balance: Butchers now have 60 in all stats to bring them slightly below carnival
tweak: Plenty of minor mapping updates
/:cl:

### CODE REVIEW

**Comment by Kitsunemitsu** [comment]
Not interested in working on city PRs at the moment.

File: ModularTegustation/tegu_items/associations/items.dm [hunk 1]
```diff
@@ -67,13 +67,21 @@
 	to_chat(user, "<span class='nicegreen'>You suddenly feel different.</span>")
 	//Guarantee one
 	user.adjust_all_attribute_levels(amount)
+	to_chat(user, "<span class='nicegreen'>You gain 1 potential!</span>")
```
Comment (LanceSmites328) [inline ModularTegustation/tegu_items/associations/items.dm:70]: This should definitely be a variable that's added up and a message is sent *once*. Also these should be using the new span wrappers.

File: ModularTegustation/tegu_items/associations/machines.dm [hunk 2]
```diff
@@ -179,6 +179,9 @@
 		if (10 - grade >= 10)
 			to_chat(user, "<span class='notice'>Potential too low to give grade. Not recommended to issue fixer license.</span>")
 			return
+		if (10 - grade <= 0)	//Once people saw Dong-Hwan, the -7 Grade fixer.
+			to_chat(user, "<span class='notice'>Recommended Grade - 1.</span>")
+			return
 
 		to_chat(user, "<span class='notice'>Recommended Grade - [10-grade].</span>")
```
Comment (LanceSmites328) [inline ModularTegustation/tegu_items/associations/machines.dm:186]:
```suggestion
		to_chat(user, "<span class='notice'>Recommended Grade - [max(10-grade, 1)].</span>")
```

File: ModularTegustation/tegu_items/associations/machines.dm [hunk 3]
```diff
@@ -179,6 +179,9 @@
 		if (10 - grade >= 10)
 			to_chat(user, "<span class='notice'>Potential too low to give grade. Not recommended to issue fixer license.</span>")
 			return
+		if (10 - grade <= 0)	//Once people saw Dong-Hwan, the -7 Grade fixer.
+			to_chat(user, "<span class='notice'>Recommended Grade - 1.</span>")
+			return
```
Comment (LanceSmites328) [inline ModularTegustation/tegu_items/associations/machines.dm:184]:
```suggestion
```

**Review by LanceSmites328 — CHANGES_REQUESTED** [review]

**Comment by Kitsunemitsu** [comment]
This broke. see 1968

**Pull request closed without merging**